plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotIntegrationSmokeTest.java

```diff
@@ -285,7 +291,7 @@ private static GenericRecord createNullRecord()
         Schema schema = getAllTypesAvroSchema();
         // Pinot does not transform the time column value to default null value
         return new GenericRecordBuilder(schema)
-                .set("updated_at", 0L)
+                .set("updated_at", initialUpdatedAt.toEpochMilli())
```

Comment:
@elonazoulay Was the setting of `updated_at` to 0 intentional here and in `createArrayNullRecord`? I don't think so but the comment `// Pinot does not transform the time column value to default null value` confused me a bit. All tests pass though.

Reply:
It was, but I didn't account for the retention. Great catch! The reason it was 0 and not null is that records with a null value for the time column do not get ingested, there's a test to verify that as well (insert nulls, verify count did not go up). This looks good to me!

Reply:
Thanks, that confirms my observation. I was a bit confused if Pinot differentiates between `0` time and `null` time or not.